test/unit/test_events.py

```diff
@@ -45,3 +46,29 @@ def test_formatting(self):
         # were no args passed after the initial message
         event = AdapterEventDebug(name="dbt_tests", base_msg="boop{x}boop", args=())
         self.assertTrue("boop{x}boop" in event.message())
+
+class TestEventCodes(TestCase):
+
+    def setUp(self):
+        pass
+
+    def get_all_subclasses(self, cls):
```

Comment:
don't forget to document this

Reply:
added documentation for get_all_subclassses, and removed setUp method as its unused currently.